content/api/hub/sample_router.md

```diff
@@ -5,24 +5,24 @@ content-type = "text/html"
 tags = ["python", "http", "routing"]
 
 [extra]
-author = "technosophos"
+author = "fermyon"
 type = "hub_document"
 category = "Sample"
 language = "Python"
 created_at = "2023-12-08T00:00:00Z"
 last_updated = "2023-12-08T00:00:00Z"
 spin_version = ">v2.0"
 summary =  "A sample that shows how to do internal HTTP routing in a Spin Python application"
-url = "https://github.com/technosophos/example-router-py"
+url = "https://github.com/fermyon/spin-python-sdk/tree/main/examples/external-lib-example"
 keywords = "router, python"
 
 ---
 
 This project showcases how to do internal HTTP routing in a Spin Python application. It is similar to [the Spin JS router](https://developer.fermyon.com/spin/v2/javascript-components#routing-in-a-component) as well as the Django router.
 
-If you are interested in extrnal routing (where the routing table is declared in the spin.toml), the [Application Structure guide](https://developer.fermyon.com/spin/v2/spin-application-structure) covers this, as does the [documentation for Spin configuration](https://github.com/technosophos/example-router-py#:~:text=the%20documentation%20for%20Spin%20configuration).
+If you are interested in extrnal routing (where the routing table is declared in the spin.toml), the [Application Structure guide](https://developer.fermyon.com/spin/v2/spin-application-structure) covers this.
```

Comment:
Also a bit disconcerting that I'm looking at an application called `external-something` and it's being _contrasted_ with `external-something-else`